1/src/components/auth/Login.vue

```diff
@@ -2,87 +2,102 @@
+              jwt
+            }
+          }
+        `,
+        // Parameters
+        variables: {
+          email: this.form.email,
+          password: this.form.password
         }
-      }`,
-      // Parameters
-      variables: {
-         email: this.form.email,
-        password: this.form.password
-      },
-    }).then((data) => {
-                    if (data.data.login != null || data.data.login!= undefined) {
-                    this.loggedIn = true;
-                    this.id = data.data.login.id;
-                    }
-                    
-                }).catch((error) => {
-                    return error;
-                })
-      },
-      // async logout(){
-      //   await this.$apollo.mutate({
-      //     mutation: gql`mutation ($id: String!) {
-      //   logout(id: $id) {
-      //     id
-      //   }
-      // }`,
-      // // Parameters
-      // variables: {
-      //    id: this.id,
-      // },
-      //   }).then((data) => {
-      //               if (data.data.logout != null || data.data.logout!= undefined) {
-      //               this.loggedIn = false;
-      //               this.id = '';
-      //               }
-                    
-      //           }).catch((error) => {
-      //               return error;
-      //           })
-      // }
+      });
+      console.log("Before if");
+      if (data.data.login != null || data.data.login != undefined) {
+        this.loggedIn = true;
+        this.id = data.data.login.id;
+      }
+      console.log("Before await");
+      const userData = await this.$apollo.query({
+        query: gql`
+          query {
+            users {
+              id
+              email
+              loggedIn
+            }
+          }
+        `
+      });
+      console.log(userData);
+    }
+    // async logout(){
+    //   await this.$apollo.mutate({
+    //     mutation: gql`mutation ($id: String!) {
+    //   logout(id: $id) {
+    //     id
+    //   }
+    // }`,
+    // // Parameters
+    // variables: {
+    //    id: this.id,
+    // },
+    //   }).then((data) => {
+    //               if (data.data.logout != null || data.data.logout!= undefined) {
+    //               this.loggedIn = false;
+    //               this.id = '';
+    //               }
+
+    //           }).catch((error) => {
+    //               return error;
+    //           })
+    // }
   },
   data() {
-      return{
-          form:{
-              email:'',
-            password:''},
-            loggedIn: false,
-            id:''
-      }
+    return {
+      form: {
+        email: "",
+        password: ""
+      },
+      loggedIn: false,
+      id: ""
+    };
   },
 
   apollo: {
     // Vue-Apollo options here
-    users: gql`query {
-      user{
-        id,
-        email,
-        # password,
-        loggedIn
+    users: gql`
+      query {
+        users {
+          id
+          email
+          # password,
+          loggedIn
```

Comment:
As I said: I find it quite suboptimal to return an attribute `loggedIn` on every user which indicates if that's the user which is logged in. Better have one mutation that's returning a `jwt` and the logged in `user`.